at_onboarding_flutter/example/lib/main.dart

```diff
@@ -15,8 +13,6 @@ void main() {
   runApp(const MyApp());
 }
 
-final StreamController<ThemeMode> updateThemeMode = StreamController<ThemeMode>.broadcast();
```

Comment:
I have a feature which allow switching theme dark/light here.
This PR will remove feature of example.
Could you please check again?